dash/include/dash/Meta.h

```diff
@@ -90,6 +90,19 @@ struct is_container_compatible :
          >
 { };
 
+/**
+ * Type trait indicating whether a type can be used for global atomic
+ * operations.
+ */
+template <typename T>
+struct is_atomic_compatible
+: public std::integral_constant<
+           bool,
+              dash::is_container_compatible<T>::value
+           && sizeof(T) <= sizeof(std::size_t)
+         >
```

Comment:
Again: the type trait `is_atomic_compatible` does **not** state that all existing atomic operations on a type are supported. It just tests if a type can be used as value type of `dash::Atomic` **at all**.
Supported atomic operations are unrelated and now tested in DASH operation types like `dash::plus` (requires arithmetic type) or `dash::second` (no restrictions).